logic/state.go

```diff
@@ -280,6 +280,7 @@ func NewStateFromConfig(config *Config) *State {
 	}
 
 	// Construct an Iceberg IO
+	log.Printf("Using Path `%v` for Iceberg Warehouse", config.Warehouse.Path)
```

Comment:
Is there a reason why we're using `printf` in all these functions as opposed to `println`? Why don't we want these logs in a new line?

Reply:
I believe `log.Printf` implicitly returns a newline at the end. Or at least, when looking at the output, thats what it does. There is no `log.Printfln` or something like that